packages/askui-nodejs/src/lib/ui-controller-args.ts

```diff
@@ -1,6 +1,24 @@
 import { LogLevels } from '../shared/log-levels';
 
+/**
+ * @param display You can choose on which display you want to excecute all tests.
+ * 0 is your main monitor.
+ * If you want to use your second monitor you can change
+ * the value to `1` (`2` for your third monitor etc.).
+ * @param binaryVersion Choose the version of the askui UI Controller Server.
+ * @param port The port the askui UI Controller is running on.
+ * @param host The host the askui UI Controller is running on.
+ * @param minimize askui UI Controller will be started as minimized window.
+ * @param overWriteBinary Download the provided Version of askui UI Controller.
+ * If a version is already downloaded. This version will be overwritten
+ * @param logLevel You can use different types of Log Level.
+ * Options: "fatal", "error", "warn", "info", "debug", "trace", "silent", "verbose".
+ * @param logFilePath It is possible to specify a path for your log files.
+ * Per default we create the askui-server.log file and askui folder in your temp folder.
+ */
```

Comment:
I though the incode description needed to be Done with [JSDoc](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html) standard. Or at least this how we did it in the file `askui/packages/askui-nodejs/src/utils/transformations.ts`